setup.py

```diff
@@ -50,6 +50,7 @@
         'Flask-CORS>=2.1.0',
     ],
     'docs': [
+        'dicttoxml>=1.7.4',
```

Comment:
This is needed because autodoc will try to load `examples/app.py` and it will fail if `dicttoxml` isn't installed.

This dependency is also described in the `requirement.txt` of the example app.